### ISSUE TITLE
Change Powertarget (Feature Request)

### ISSUE THREAD

**Issue by Simon0Harms** [ecs87/AMD-GPU-Tool#3]
Are I am right that at the moment you can't change the powertarget with this tool?

**Comment by ecs87** [comment]
Yes, you'd be correct. The main reason I did not add it was to force users
to properly manage their power consumption by modifying voltage at various
power levels.

By just changing the power target, you're losing out on clock speeds and
still using more energy than properly managing power consumption through
voltage.

On Dec 7, 2017 6:19 AM, "Simon0Harms" <notifications@github.com> wrote:

Are I am right that at the moment you can't change the powertarget with
this tool?

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/ecs87/AMD-GPU-Tool/issues/3>, or mute the thread

**Comment by Simon0Harms** [comment]
Yeah, i like the powertarget as lazy possibillity.
And when i Reduce the Mhz of the GPU to less then 1200 it is getting unstable. (With the Stock AMD Tool)
But  when i use the powertarget it works, and the Card downclock to 11XX Mhz....

Never the less changing der Mhz with your tool don't work for me either. (Tried it a month ago)
He changes the Mhz to the "save state" 500 Mhz (Mem Clock) instead of 9XX....

Vega56
